README.md

```diff
@@ -1,3 +1,49 @@
 # sellotape
-## Team Members
-Guy Itzhaki, Tarel Madar, Gal Lapid, Alon weissfeld, Omri Rosner
+
+Sellotape is a modern audio podcast platform.
+It introduces an accessible way to listen to podcast and audio streams.
+
+As a podcaster/streamer, you can create your private radio. By accessing your user endpoint, anyone who wants to listen can join your podcast, live djset, or whatever it is you want it to be heard.
+
+As a listener, you can follow your favorite streamers and keep up to date with their streams.
+
+## Ideology 
+The platform strives to be a simple social network as users can follow each other and go live whenever they wish to do so. Our philosophy is that there should be no alienation between a streamer and a listener, and we engage and strive to push the interaction between them as possible.
+
+## High-level design
+* Login/register page
```

Comment:
how about some kind of an interesting landing page for unregistered users?

I suggest you try to describe your application in the terms of the kinds of users that use it and their workflows, instead of describing the individual pages. . This is more along the lines of what you wrote in the introduction above, only with a bit more detail. Such a description can be easily turned later into user stories.